app/models/card.rb

```diff
@@ -1,12 +1,22 @@
 class Card < ActiveRecord::Base
 
-  validates :original_text, presence: true
+  validates :original_text,   presence: true
+  validates :translated_text, presence: true
+  validates :review_date,     presence: true
+  validate  :check_duplication
+
 
   before_create do
     set_default_date
   end
 
+  def check_duplication
+    if original_text.downcase == translated_text.downcase
+      errors.add(:duplication, 'Значения полей дублируют друг друга.') 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.